flask_calendar/constants.py

```diff
@@ -0,0 +1,3 @@
+
+SESSION_ID = "sid"
+TEMPLATE_DIR = "../templates"
```

Comment:
Relative imports that traverse to parent folders are less secure than relative imports, I'd rather rely on having specified a correct `PYTHONPATH` or initializing Flask to get `flask_calendar` as the base folder instead

Reply:
I'm moving `static/` and `templates/` inside the new `flask_calendar` package, where it seems to be recommended in Flask (see [1] and [2]). Is this OK for you?

[1]: https://flask.palletsprojects.com/en/1.1.x/quickstart/#rendering-templates
[2]: https://exploreflask.com/en/latest/templates.html#how-to-organize-templates

Reply:
yes please, perfect!